_docs/index.md

```diff
@@ -30,12 +30,12 @@ The goals of the <b>U.S. IOOS National Glider Data Assembly Center</b>:
+The **NGDAC** accepts a [simple NetCDF file](/glider-dac/ngdac-netcdf-file-format-version-2.html) containing water column measurements collected by a glider during a single profile (see glider sampling terminology [here](glider-background-and-sampling-terminology.html#sampling-pattern-terminology)).  Groups of these NetCDF files, gathered during a deployment (also known as a [**trajectory**](glider-background-and-sampling-terminology.html#sampling-pattern-terminology)), are uploaded to the **NGDAC** by individual glider operators.  Once they arrive at the **NGDAC**, the files are validated for compliance, aggregated into a single dataset representing the **deployment/trajectory** and distributed via [ERDDAP](https://coastwatch.pfeg.noaa.gov/erddap/information.html) and [THREDDS](https://www.unidata.ucar.edu/software/tds/) end-points.  The data sets served by the **NGDAC** provide access to the **trajectory/deployment** data both as time-series and on a profile-by-profile basis.
 
-Transmission of the data sets via the [Global Telecommunication System](http://www.wmo.int/pages/prog/www/TEM/GTS/index_en.html) is made possible by the [National Data Buoy Center](http://www.ndbc.noaa.gov/), which accesses the data sets from the **ERDDAP** and/or **THREDDS** end-points. After performing some internal quality checks, the profiles are encoded into [BUFR](http://en.wikipedia.org/wiki/BUFR) format and released on the [Global Telecommunication System](http://www.wmo.int/pages/prog/www/TEM/GTS/index_en.html), making them available for assimilation by regional and global scale ocean forecasting models.
+Transmission of the data sets via the [Global Telecommunication System](https://public.wmo.int/en/programmes/global-telecommunication-system) is made possible by the [National Data Buoy Center](http://www.ndbc.noaa.gov/), which accesses the data sets from the **ERDDAP** and/or **THREDDS** end-points. After performing some internal quality checks, the profiles are encoded into [BUFR](http://en.wikipedia.org/wiki/BUFR) format and released on the [Global Telecommunication System](https://public.wmo.int/en/programmes/global-telecommunication-system), making them available for assimilation by regional and global scale ocean forecasting models.
 
 Please read the documentation **thoroughly** before beginning the data submission process.  Additional questions and information requests should be directed to:
```

Comment:
Delete the sentence "Please read documentation **thoroughly**..."

Reply:
Done!